apps/system/js/attention_window_manager.js

```diff
@@ -241,7 +240,6 @@
           }
           this._openedInstances.delete(attention);
           this.updateAttentionIndicator();
-          this.updateClassState();
```

Comment:
I think we need to check `attention.isCrashed` here and still call `updateClassState()` if true because in case of an OOM kill / a crash we might not get the `attentionclosed` event and leave the screen in `attention` mode.

And of course we should add a test for this :)